### ISSUE TITLE
"db service health check timed out" w/Drupal 8

### ISSUE THREAD

**Issue by rickmanelius** [drud/ddev#510]
## What happened (or feature request):

* Bug

## What you expected to happen:

As per an active conversation in the #ddev room within the Drupal Slack channel (https://drupal.slack.com/archives/C5TQRQZRR/p1508188921000097), @brentrobbins is having an issue with spinning up a Drupal 8.4 site and he can replicate the following error: "db service health check timed out"

## How to reproduce this:

We need to recreate it internally.

## Version: Please include the output of `ddev version`, `docker version` and the project's .ddev/config.yaml.

ddev v0.9.3

**Comment by rickmanelius** [comment]
Things already tested:
* Docker reset
* Memory limit increase from 2GB to 3GB

And what appears to be the source of the timeout. 
```
ERROR 2013 (HY000) at line 16837: Lost connection to MySQL server during query
/entrypoint.sh: line 96:    49 Killed                  mysqld --skip-networking
```

**Comment by brentrobbins** [comment]
Hey @rickmanelius, the issues I am experiencing are with (existing) Drupal 7 (it worked with a new D7 site), however it didn't work with a new D8 site as well.

**Comment by rickmanelius** [comment]
This may be related to this issue https://github.com/drud/ddev/issues/449, but this was more of a web mount not properly working or being available after a `ddev start`

**Comment by rickmanelius** [comment]
After working through the issue in the Drupal community Slack, we uncovered a few things. First, we're excited that @brentrobbins was attempting to run 7 sites concurrently! Additionally, there were other left over docker containers as well as a few Solr containers. Altogether, this was causing a resourcing issue with Docker and preventing mysql from within the mysql container from being able to stay up. We worked through this and now he's operational.

We'll file a separate issue related to warning users and providing guidance to them when they are pushing Docker to the limits of the available resources. We can nudge said users during a new `ddev start` or when more than 3 sites appear in a `ddev list` command.
